netlify.toml

```diff
@@ -0,0 +1,5 @@
+[template.environment]
+  WHITELISTED_ADDRESSES=0x692a70D2e424a56D2C6C27aA97D1a86395877b3A 0xa88c953d9bFcdcFa46d9ec29586820D55C4EC0dC
+  WHITELISTED_METHODS=vote(uint8) test(address,uint8)
+  WALLET_MNEMONIC="your mnemonic" 
+  NETWORK_URL=http://127.0.0.1:8545
```

Comment:
For clarity sake, since this is going to be used on a remote server, should we set this to a infura mainnet endpoint?

Reply:
You're right sir. Going to update this NETWORK_URL env variable to an infura mainnet endpoint